state-chain/runtime/src/lib.rs

```diff
@@ -2009,9 +2022,15 @@ impl_runtime_apis! {
 			pallet_cf_environment::RuntimeSafeMode::<Runtime>::get()
 		}
 
-		fn cf_pools() -> Vec<PoolPairsMap<Asset>> {
+		fn cf_pool_pairs() -> Vec<PoolPairsMap<Asset>> {
 			LiquidityPools::pools()
 		}
+
+		fn cf_pools() -> BTreeMap<AssetPair, Pool<Runtime>> {
+
+			pallet_cf_pools::Pools::<Runtime>::iter().collect()
+
```

Comment:
This is a (potentially quite confusing) breaking change. 

It looks like this is just a workaround to access the pools storage - we can access it directly by adding a reference to the state backend to the rpc provider, and avoid adding another api method we will need to maintain.